Print_A_Bot/djwifi/forms.py

```diff
@@ -0,0 +1,9 @@
+from django import forms
+from django.utils.translation import ugettext as _
+from djconfig.forms import ConfigForm
+
+
+class WifiForm(ConfigForm):
```

Comment:
this is used to save 3 "settings" into the db so they can be changed relatively easily